lib/modules/uscore_v3.1.1/bodyheight_sequence.rb

```diff
@@ -286,33 +286,39 @@ def patient_ids
         patient_ids.each do |patient|
           next unless @observation_ary[patient].present?
 
-          search_params = {
-            'patient': patient,
-            'category': get_value_for_search_param(resolve_element_from_path(@observation_ary[patient], 'category') { |el| get_value_for_search_param(el).present? }),
-            'date': get_value_for_search_param(resolve_element_from_path(@observation_ary[patient], 'effective') { |el| get_value_for_search_param(el).present? })
-          }
+          observation_ary = Array.wrap(@observation_ary[patient])
```

Comment:
I think having `observation_ary` and `@observation_ary` will be pretty confusing. Could we use `observations` or `patient_observations` instead? I'm pretty sure I checked a while ago and just adding an `s` works to pluralize all the resources we care about.

Reply:
Combined this line with next line to
```
Array.wrap(@observation_ary[patient]).each do |observation|
```